src/sync/commits.ts

```diff
@@ -26,14 +23,16 @@ export const getCommitTask = async (
 	logger: LoggerWithTarget,
 	_github: GitHubAPI,
 	gitHubClient: GitHubInstallationClient,
-	_jiraHost: string,
+	jiraHost: string,
 	repository: Repository,
 	cursor?: string | number,
-	perPage?: number): Promise<CommitData> => {
-	logger.info("Syncing commits: started");
-	const { edges, commits } = await fetchCommits(gitHubClient, repository, cursor, perPage);
+	perPage?: number): Promise<TaskPayload<CommitQueryNode, JiraCommitData>> => {
+
+	logger.debug("Syncing commits: started");
+	const timeCutoff = await numberFlag(NumberFlags.SYNC_MAIN_COMMIT_TIME_LIMIT, NaN, jiraHost);
```

Comment:
Shouldn't the default value be `undefined`, not NaN? (Since you are passing it over later as an optional parameter)